test/ci/buildkite/ngtf-ie-modeltest.yml

```diff
@@ -0,0 +1,42 @@
+  - command: |
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      python3 build_ngtf.py ${BUILD_OPTIONS} \
+      --artifacts /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID \
+      --use_tensorflow_from_location /localdisk/buildkite-agent/prebuilt_tensorflow_2_2_0
+      pip install -U /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/tensorflow/tensorflow-2.2.0-cp36-cp36m-linux_x86_64.whl
+      pip install -U /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/ngraph_tensorflow_bridge-*.whl
```

Comment:
Why is pip install required if building from source?

Reply:
To update the venv after build with TF and NGTF. Required. Otherwise the later step will fail looking for those pip modules.